.github/workflows/ci-pytorch-dockers.yml

```diff
@@ -94,12 +93,9 @@ jobs:
         include:
           # These are the base images for PL release docker images,
           # so include at least all of the combinations in release-dockers.yml.
-          - {python_version: "3.9", pytorch_version: "1.9", cuda_version: "11.1.1"}
           - {python_version: "3.9", pytorch_version: "1.10", cuda_version: "11.3.1"}
           - {python_version: "3.9", pytorch_version: "1.11", cuda_version: "11.3.1"}
           - {python_version: "3.9", pytorch_version: "1.12", cuda_version: "11.6.1"}
-          # Used in Lightning-AI/tutorials
-          - {python_version: "3.8", pytorch_version: "1.9", cuda_version: "11.1.1"}
```

Comment:
Was the comment correct? Is it really used in tutorials? If so, shouldn't that be also updated?

Reply:
Updated! https://github.com/Lightning-AI/tutorials/pull/203